sources/LLKEY

```diff
@@ -1754,13 +1752,13 @@
         (74 ("Function,LF" "Function,LF" NOLOCKSHIFT))
         (75 ("^B" "^B" NOLOCKSHIFT))
         (81 ("Function,FF" "7" NOLOCKSHIFT))
-        (82 ("#-" "8" NOLOCKSHIFT))
+        (UPARROW ("Meta,^" "8" NOLOCKSHIFT))
```

Comment:
No other keyactions are specified by key name - should these not be the key number?